starboard/nplb/posix_compliance/posix_socket_errno_test.cc

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#if SB_API_VERSION >= 16
```

Comment:
Doesn't errno not work in EG SB15 and SB14? 
This test will fail for EG SB15 and SB14, errno will always return "No error information".

Reply:
for example: https://github.com/youtube/cobalt/actions/runs/8638213306/job/23682492622?pr=2648

Reply:
Let's try to make it work otherwise Cobalt would have to do a lot of #ifdef checks and errno usage wouldn't be that useful.

I think we should try to define 
```
#if SB_API_VERSION < 16
#define errno SbSystemGetLastError()
#endif
```